app/assets/javascripts/map.js

```diff
@@ -74,26 +74,33 @@ require([
      * you are not completely sure.
      */
     _initViews: function() {
-      // Google Experiments
-      new ExperimentsPresenter();
-
-      var mapView = new MapView();
-
-      new MapControlsView(mapView.map);
-      new TabsView(mapView.map);
-      new AnalysisResultsView();
-      new LayersNavView();
-      new LegendView();
-      new TimelineView();
-      new NavMobileView();
-      new FooterView();
-      new HeaderView();
-      new SourceModalView();
-      new SourceBottomView();
-      new SourceMobileFriendlyView();
-      new NotificationsView();
-      new GuideView();
-      new GuideButtonView();
+
+      var map = new MapView();
+      new MapControlsView(this.map, this.countries);
+
+      this.map = map.map;
+      this.countries = new CountryCollection();
+
+      // Init views
+      this.countries.fetch().done(function(){
```

Comment:
Ah ok, maybe in that case you can get away with depending on `countries.fetch()` but pull out some of the views that don't depend on the collection so that most of the UI loads while the countries are being fetched.